cypress/support/hostsfunctions.ts

```diff
@@ -239,3 +253,30 @@ export function testHostBulkDelete(host_type: string, inventory: Inventory) {
     );
   }
 }
+
+export function testMissingButton(
+  host_type: string,
+  inventory: Inventory,
+  view: string,
+  missing: string
+) {
+  //navigate to list view and check host existing
+  //in case this should be done to detailes view the only thing needed is to add click on host name
+  navigateToBaseView(host_type, inventory.name);
+  cy.get(`[aria-label="Simple table"] tr`).should('have.length', 2);
+  cy.get(missing).should('not.exist');
+}
+
+export function testMissingTab(
+  host_type: string,
+  inventory: Inventory,
+  view: string,
+  missing: string
+) {
+  //navigate to list view and check host existing
+  //get to host to verify tab is missing
+  navigateToBaseView(host_type, inventory.name);
+  cy.get(`[aria-label="Simple table"] tr`).should('have.length', 2);
+  cy.getByDataCy('name-column-cell').contains('E2E Host').click();
+  cy.contains('[role="tab"]', missing).should('not.exist');
```

Comment:
suggestion in place of negative assertion, can we assert on what should exist instead?

Reply:
that's just nothing in this place